script/Ciel/Day 4/04_01_CIEL04_6_2.txt

```diff
@@ -37,7 +37,7 @@ She forgave Tohno Shiki's unforgivable sin with a smile on her face.
 [sha:4abae63caa299c818bd3f5ec65f3dc7a8ecf9d33]{
 -- Page 143, Offset 8537.
 -- なら―――自分の命ぐらい笑って差し出すぐらいはしなくちゃ人間として恥ずかしい。
-If she can do that―――then I should be ashamed of myself as a human being if I don't return that smile as I offer up my own life.
+If she can do that――then I should be ashamed of myself as a human being if I don't return that smile as I offer up my own life.
```

Comment:
dash not needed > comma

though I almost wish this was rewritten or something

but who cares rn